### ISSUE TITLE
ipmctl -help not supported

### ISSUE THREAD

**Issue by kjlyon** [intel/ipmctl#83]
`ipmctl -h` and `ipmctl -help` return syntax error stating that `-h` or `-help` are not supported verbs. Though, `ipmctl help` works as expected, as does `ipmctl show -h`. This is not a huge deal, but it does clash with the documentation which states:
```
Display the CLI help.
    help [-help|-h] [-output|-o (text|nvmxml)]
```

**Comment by gldiviney** [comment]
I think we've addressed this. Where in the documentation did you see this? If it was just from the CLI, can you try again with the latest? thanks!

**Comment by sscargal** [comment]
The problem is the order of verb and options which is currently quite strict.  For example:

```
Usage:
# ipmctl 
Display the CLI help.
    help [-help|-h] [-output|-o (text|nvmxml)]
[...]
```

If we specify the verb then the options, as documented, it works as expected.  But if we put the `-h` or `--help` before the verb, you get the syntax error.

```
# ipmctl help -h

Display the CLI help.
    help [-help|-h] [-output|-o (text|nvmxml)]

# ipmctl -h help

Syntax Error: First token must be a verb, '-h' is not a supported verb.


# ipmctl help -help

Display the CLI help.
    help [-help|-h] [-output|-o (text|nvmxml)]

# ipmctl -help help

Syntax Error: First token must be a verb, '-help' is not a supported verb.
```

At least for the `-h/--help` option we should consider relaxing the ordering since it's generic and many other Linux commands permit putting `-h/--help` in any order.  For example, both of these work without returning a syntax error:

```
# git --help branch
# git branch --help
```

**Comment by gldiviney** [comment]
Fixed: 0538a3d25449869f32bc999a366dc5bad1740870
